h2o-docs/src/product/starting-h2o.rst

```diff
@@ -279,13 +301,13 @@ H2O Options
     - IPv4: ``-network 178.0.0.0/8``
     - IPv6: ``-network 2001:db8:1234:0:0:0:0:0/48`` (short version of IPv6 with ``::`` is not supported.)
 
--	``-ice_root <fileSystemPath>``: Specify a directory for H2O to spill temporary data to disk (where ``<fileSystemPath>`` is the file path).
-- ``-log_dir <fileSystemPath>``: Specify the directory where H2O writes logs to disk. (This usually has a good default that you need not change.
+-	``-ice_root <fileSystemPath>``: Specify a directory for H2O-3 to spill temporary data to disk (where ``<fileSystemPath>`` is the file path).
+- ``-log_dir <fileSystemPath>``: Specify the directory where H2O-3 writes logs to disk. (This usually has a good default that you need not change.
 - ``-log_level <TRACE,DEBUG,INFO,WARN,ERRR,FATAL>``: Specify to write messages at this logging level, or above. The default is INFO.
 - ``-flow_dir <server-side or HDFS directory>``: Specify a directory for saved flows. The default is ``/Users/h2o-<H2OUserName>/h2oflows`` (where ``<H2OUserName>`` is your user name).
-- ``-file_deny_glob <GLOB pattern>``: Specify `glob <https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob>`_. pattern to deny access to certain files. The default is ``{/bin/*,/etc/*,/var/*,/usr/*,/proc/*,**/.**}``.
+- ``-file_deny_glob <GLOB pattern>``: Specify the `glob <https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob>`__ pattern to deny access to certain certain directories. This parameter is added to remove vulnerabilities CVE-2023-6038, CVE-2023-6569 and CVE-2024-5986. The default is ``{/bin/*,/etc/*,/var/*,/usr/*,/proc/*,**/.**}``.
```

Comment:
Please change

deny access to certain certain directories.

To

deny access to certain directories.